sources/packages/backend/apps/queue-consumers/src/processors/assessment/cancel-application-assessment.processor.ts

```diff
@@ -58,6 +58,20 @@ export class CancelApplicationAssessmentProcessor {
       throw new Error(errorMessage);
     }
 
+    if (
+      assessment.studentAssessmentStatus !==
+      StudentAssessmentStatus.CancellationQueued
+    ) {
+      await job.discard();
+      const warnMessage = `Assessment id ${job.data.assessmentId} is not in ${StudentAssessmentStatus.CancellationQueued} status.`;
+      this.logger.warn(warnMessage);
+      summary.warn(warnMessage);
+      summary.info(
+        "Workflow process not executed due to the assessment cancellation not being in the correct status.",
```

Comment:
"Workflow process not executed" can be misleading since the workflow is being canceled.
Can we please adjust to "Workflow cancellation process not executed"?